api/src/middleware/login-validator.ts

```diff
@@ -1,11 +1,14 @@
 import { Request, Response, NextFunction } from "express";
+import { createError } from "../utils/errors";
 
 // Checks if user already has a session
 export function checkNotAuthenticated (req: Request, res: Response, next: NextFunction) {
   if (!req.user) {
     next();
   } else {
     // Redirect to a profile page as user is already logged in
-    res.status(400).send({ message: "There is already a session. You need to log out."}); // Placeholder
+    res.status(400).send({ errors: [ {
```

Comment:
Is there a reason to give an error here? Like could there be a situation where the client would need to be aware that they already have a session or would it be better to just bypass and respond 200/OK so as far as the client knows they've authed successfully?